<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MusicApplication</class>
 <widget class="QWidget" name="MusicApplication">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>990</width>
    <height>660</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>天天酷音</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../musicQrc/MusicPlayer.qrc">
    <normaloff>:/image/windowicon</normaloff>:/image/windowicon</iconset>
  </property>
  <widget class="QToolButton" name="windowClose">
   <property name="geometry">
    <rect>
     <x>937</x>
     <y>27</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QToolButton" name="minimization">
   <property name="geometry">
    <rect>
     <x>909</x>
     <y>27</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="background">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>950</width>
     <height>620</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="musicPrivious">
   <property name="geometry">
    <rect>
     <x>114</x>
     <y>80</y>
     <width>55</width>
     <height>55</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QToolButton" name="musicNext">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>80</y>
     <width>55</width>
     <height>55</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="MusicTimerSliderWidget" name="musicTimeWidget" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>45</y>
     <width>375</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QToolButton" name="menuSetting">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>600</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="popupMode">
    <enum>QToolButton::InstantPopup</enum>
   </property>
  </widget>
  <widget class="QToolButton" name="musicKey">
   <property name="geometry">
    <rect>
     <x>177</x>
     <y>80</y>
     <width>55</width>
     <height>55</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QToolButton" name="musicSound">
   <property name="geometry">
    <rect>
     <x>303</x>
     <y>114</y>
     <width>18</width>
     <height>18</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/image/soundmax);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QSlider" name="musicSoundSlider">
   <property name="geometry">
    <rect>
     <x>328</x>
     <y>116</y>
     <width>53</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QStackedWidget" name="songsContainer">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>138</y>
     <width>331</width>
     <height>461</height>
    </rect>
   </property>
  </widget>
  <widget class="MusicMarqueeWidget" name="showCurrentSong" native="true">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>30</y>
     <width>200</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#FFFFFF;font-weight:bold;font-size:15px;</string>
   </property>
   <property name="text" stdset="0">
    <string>当前无正在播放的歌曲</string>
   </property>
  </widget>
  <widget class="QToolButton" name="musicSetting">
   <property name="geometry">
    <rect>
     <x>165</x>
     <y>600</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QToolButton" name="musicImport">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>600</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QToolButton" name="musicSearch">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>600</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QToolButton" name="musicCurrentLocation">
   <property name="geometry">
    <rect>
     <x>295</x>
     <y>600</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="MusicLocalSongSearchInlineEdit" name="musicSongSearchLine" native="true">
   <property name="geometry">
    <rect>
     <x>504</x>
     <y>27</y>
     <width>261</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text" stdset="0">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="musicWindowSpace">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>20</y>
     <width>590</width>
     <height>45</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/image/screentop);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="musicSearchButton">
   <property name="geometry">
    <rect>
     <x>765</x>
     <y>27</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/share/searchlineleft);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QToolButton" name="musicWindowChangeSkin">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>27</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QStackedWidget" name="SurfaceStackedWidget">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>144</y>
     <width>521</width>
     <height>455</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QLabel" name="showPicture">
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/beauty);</string>
    </property>
   </widget>
   <widget class="MusicSongSearchOnlineWidget" name="songSearchWidget"/>
   <widget class="MusicLrcContainerForInline" name="musiclrccontainerforinline"/>
  </widget>
  <widget class="QPushButton" name="musicSearchWidgetButton">
   <property name="geometry">
    <rect>
     <x>595</x>
     <y>88</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>音乐</string>
   </property>
  </widget>
  <widget class="QPushButton" name="musicLrcWidgetButton">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>88</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>歌词</string>
   </property>
  </widget>
  <widget class="QPushButton" name="musicSearchRefreshButton">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>90</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="musicIndexWidgetButton">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>88</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>首页</string>
   </property>
  </widget>
  <widget class="QLabel" name="showDownloadGif">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>510</y>
     <width>41</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="musicDesktopLrc">
   <property name="geometry">
    <rect>
     <x>35</x>
     <y>113</y>
     <width>15</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="musicBestLove">
   <property name="geometry">
    <rect>
     <x>87</x>
     <y>108</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>26</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="userWindow">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
   </layout>
  </widget>
  <widget class="QToolButton" name="musicButton_tools">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>475</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="popupMode">
    <enum>QToolButton::InstantPopup</enum>
   </property>
  </widget>
  <widget class="MusicEnhancedWidget" name="musicEnhancedButton" native="true">
   <property name="geometry">
    <rect>
     <x>335</x>
     <y>95</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolButton" name="musicWindowRemote">
   <property name="geometry">
    <rect>
     <x>878</x>
     <y>27</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QToolButton" name="musicButton_playlist">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>175</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QToolButton" name="musicButton_radio">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>235</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QToolButton" name="musicButton_mydownl">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>295</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QToolButton" name="musicButton_cloud">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>415</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="vedioWidgetButton">
   <property name="geometry">
    <rect>
     <x>785</x>
     <y>88</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>视频</string>
   </property>
  </widget>
  <widget class="QLabel" name="playCurrentTime">
   <property name="geometry">
    <rect>
     <x>316</x>
     <y>75</y>
     <width>30</width>
     <height>15</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#FFFFFF;</string>
   </property>
   <property name="text">
    <string>00:00</string>
   </property>
  </widget>
  <widget class="QLabel" name="playTotalTime">
   <property name="geometry">
    <rect>
     <x>347</x>
     <y>75</y>
     <width>35</width>
     <height>15</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#FFFFFF;</string>
   </property>
   <property name="text">
    <string>/00:00</string>
   </property>
  </widget>
  <widget class="QToolButton" name="musicPlayMode">
   <property name="geometry">
    <rect>
     <x>56</x>
     <y>108</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QToolButton" name="musicWindowConcise">
   <property name="geometry">
    <rect>
     <x>848</x>
     <y>27</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="musicWindowSpace_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>601</y>
     <width>951</width>
     <height>39</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgba(0,0,0,30)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="resizeWindowLabel">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>620</y>
     <width>15</width>
     <height>15</height>
    </rect>
   </property>
   <property name="pixmap">
    <pixmap resource="../../musicQrc/MusicPlayer.qrc">:/image/resizeWindow</pixmap>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>35</x>
     <y>35</y>
     <width>46</width>
     <height>21</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="musicQualityWindow"/>
  </widget>
  <widget class="QToolButton" name="musicButton_mobile">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>355</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="lrcDisplayAllButton">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>392</x>
     <y>320</y>
     <width>15</width>
     <height>56</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{border:none;background:transparent;};</string>
   </property>
  </widget>
  <widget class="QPushButton" name="musicSearchBackButton">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>90</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <action name="action_About">
   <property name="text">
    <string>关于天天酷音(&amp;A)</string>
   </property>
   <property name="toolTip">
    <string>关于天天酷音</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="action_Quit">
   <property name="icon">
    <iconset resource="../../musicQrc/MusicPlayer.qrc">
     <normaloff>:/image/close</normaloff>:/image/close</iconset>
   </property>
   <property name="text">
    <string>退出(&amp;Q)</string>
   </property>
   <property name="toolTip">
    <string>退出</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action_ImportSongs">
   <property name="icon">
    <iconset resource="../../musicQrc/MusicPlayer.qrc">
     <normaloff>:/appTools/import</normaloff>:/appTools/import</iconset>
   </property>
   <property name="text">
    <string>导入本地歌曲(&amp;I)</string>
   </property>
   <property name="toolTip">
    <string>添加歌曲</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="action_Privious">
   <property name="icon">
    <iconset resource="../../musicQrc/MusicPlayer.qrc">
     <normaloff>:/image/privious</normaloff>:/image/privious</iconset>
   </property>
   <property name="text">
    <string>上一曲(&amp;P)</string>
   </property>
   <property name="toolTip">
    <string>上一曲</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Left</string>
   </property>
  </action>
  <action name="action_Next">
   <property name="icon">
    <iconset resource="../../musicQrc/MusicPlayer.qrc">
     <normaloff>:/image/next</normaloff>:/image/next</iconset>
   </property>
   <property name="text">
    <string>下一曲(&amp;N)</string>
   </property>
   <property name="toolTip">
    <string>下一曲</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Right</string>
   </property>
  </action>
  <action name="action_Play">
   <property name="icon">
    <iconset resource="../../musicQrc/MusicPlayer.qrc">
     <normaloff>:/image/play</normaloff>:/image/play</iconset>
   </property>
   <property name="text">
    <string>播放/暂停</string>
   </property>
   <property name="toolTip">
    <string>播放/暂停</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="action_VolumePlus">
   <property name="icon">
    <iconset resource="../../musicQrc/MusicPlayer.qrc">
     <normaloff>:/image/soundmax</normaloff>:/image/soundmax</iconset>
   </property>
   <property name="text">
    <string>音量(&amp;+)</string>
   </property>
   <property name="toolTip">
    <string>增加音量</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Up</string>
   </property>
  </action>
  <action name="action_VolumeSub">
   <property name="icon">
    <iconset resource="../../musicQrc/MusicPlayer.qrc">
     <normaloff>:/image/soundmin</normaloff>:/image/soundmin</iconset>
   </property>
   <property name="text">
    <string>音量(&amp;-)</string>
   </property>
   <property name="toolTip">
    <string>降低音量</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Down</string>
   </property>
  </action>
  <action name="action_OrderPlay">
   <property name="icon">
    <iconset resource="../../musicQrc/MusicPlayer.qrc">
     <normaloff>:/image/orderplay</normaloff>:/image/orderplay</iconset>
   </property>
   <property name="text">
    <string>顺序播放</string>
   </property>
   <property name="toolTip">
    <string>顺序播放</string>
   </property>
   <property name="shortcut">
    <string>Alt+1</string>
   </property>
  </action>
  <action name="action_RandomPlay">
   <property name="icon">
    <iconset resource="../../musicQrc/MusicPlayer.qrc">
     <normaloff>:/image/randomplay</normaloff>:/image/randomplay</iconset>
   </property>
   <property name="text">
    <string>随机播放</string>
   </property>
   <property name="toolTip">
    <string>随机播放</string>
   </property>
   <property name="shortcut">
    <string>Alt+2</string>
   </property>
  </action>
  <action name="action_SingleCycle">
   <property name="icon">
    <iconset resource="../../musicQrc/MusicPlayer.qrc">
     <normaloff>:/image/oneloop</normaloff>:/image/oneloop</iconset>
   </property>
   <property name="text">
    <string>单曲循环</string>
   </property>
   <property name="toolTip">
    <string>单曲循环</string>
   </property>
   <property name="shortcut">
    <string>Alt+3</string>
   </property>
  </action>
  <action name="action_ListCycle">
   <property name="icon">
    <iconset resource="../../musicQrc/MusicPlayer.qrc">
     <normaloff>:/image/listloop</normaloff>:/image/listloop</iconset>
   </property>
   <property name="text">
    <string>列表循环</string>
   </property>
   <property name="toolTip">
    <string>列表循环</string>
   </property>
   <property name="shortcut">
    <string>Alt+4</string>
   </property>
  </action>
  <action name="action_Setting">
   <property name="icon">
    <iconset resource="../../musicQrc/MusicPlayer.qrc">
     <normaloff>:/appTools/setting</normaloff>:/appTools/setting</iconset>
   </property>
   <property name="text">
    <string>设置(&amp;S)</string>
   </property>
   <property name="toolTip">
    <string>设置</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <zorder>background</zorder>
  <zorder>musicPrivious</zorder>
  <zorder>musicNext</zorder>
  <zorder>musicKey</zorder>
  <zorder>musicSound</zorder>
  <zorder>musicSoundSlider</zorder>
  <zorder>songsContainer</zorder>
  <zorder>showCurrentSong</zorder>
  <zorder>musicWindowSpace</zorder>
  <zorder>musicSongSearchLine</zorder>
  <zorder>musicSearchButton</zorder>
  <zorder>musicWindowChangeSkin</zorder>
  <zorder>SurfaceStackedWidget</zorder>
  <zorder>musicSearchWidgetButton</zorder>
  <zorder>musicLrcWidgetButton</zorder>
  <zorder>musicSearchRefreshButton</zorder>
  <zorder>musicIndexWidgetButton</zorder>
  <zorder>showDownloadGif</zorder>
  <zorder>musicDesktopLrc</zorder>
  <zorder>musicBestLove</zorder>
  <zorder>horizontalLayoutWidget</zorder>
  <zorder>musicButton_tools</zorder>
  <zorder>musicWindowRemote</zorder>
  <zorder>musicButton_playlist</zorder>
  <zorder>musicButton_radio</zorder>
  <zorder>musicButton_mydownl</zorder>
  <zorder>musicButton_cloud</zorder>
  <zorder>musicTimeWidget</zorder>
  <zorder>vedioWidgetButton</zorder>
  <zorder>playCurrentTime</zorder>
  <zorder>playTotalTime</zorder>
  <zorder>musicPlayMode</zorder>
  <zorder>musicWindowConcise</zorder>
  <zorder>musicWindowSpace_2</zorder>
  <zorder>musicSetting</zorder>
  <zorder>menuSetting</zorder>
  <zorder>musicCurrentLocation</zorder>
  <zorder>musicImport</zorder>
  <zorder>musicSearch</zorder>
  <zorder>resizeWindowLabel</zorder>
  <zorder>minimization</zorder>
  <zorder>verticalLayoutWidget</zorder>
  <zorder>musicEnhancedButton</zorder>
  <zorder>musicButton_mobile</zorder>
  <zorder>lrcDisplayAllButton</zorder>
  <zorder>windowClose</zorder>
  <zorder>musicSearchBackButton</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MusicMarqueeWidget</class>
   <extends>QWidget</extends>
   <header>musicmarqueewidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MusicLrcContainerForInline</class>
   <extends>QWidget</extends>
   <header>musiclrccontainerforinline.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MusicSongSearchOnlineWidget</class>
   <extends>QWidget</extends>
   <header>musicsongsearchonlinewidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MusicTimerSliderWidget</class>
   <extends>QWidget</extends>
   <header>musictimersliderwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MusicLocalSongSearchInlineEdit</class>
   <extends>QWidget</extends>
   <header>musiclocalsongsearchinlineedit.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MusicEnhancedWidget</class>
   <extends>QWidget</extends>
   <header>musicenhancedwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../musicQrc/MusicPlayer.qrc"/>
 </resources>
 <connections/>
</ui>
